### PR TITLE
Feature/concept

### PR DESCRIPTION
**Pull request by unitatem**

### CODE REVIEW

File: src/main.R [hunk 1]
```diff
@@ -1,13 +1,97 @@
...
+read_csv_file = function(path) {
+    data = read.csv(file = path,
+                    header = TRUE,
+                    sep = ",")
+    return(data)
+}
 
-resources_path = "../resources/"
-credit_card_file = "creditcard.csv"
+extract_noraml_anomaly = function(mixed_data) {
```
Comment (Verntrax) [inline src/main.R]: extract_normal_anomaly

File: src/main.R [hunk 2]
```diff
@@ -1,13 +1,97 @@
...
+    for (i in 1:nrow(params$centers)) {
+        distance = dist(rbind(element, params$centers[i, ]))
+        if (distance <= params$distance[i])
+            return(TRUE)
+    }
+    return(FALSE)
+}
+
+calculate_success = function(data_test, clusters_param) {
+    correct_decition = list(true_anomaly = 0, true_normal = 0)
```
Comment (Verntrax) [inline src/main.R]: correct_decision

File: src/main.R [hunk 3]
```diff
@@ -1,13 +1,97 @@
...
+    perm_vect = sample(1:normal_anomaly$normal_nrow)
+    divide_idx = nrow(normal_anomaly$anomaly)
+
+    idx_normal_2_test = perm_vect[1:divide_idx]
+    idx_normal_2_training = perm_vect[(divide_idx + 1):normal_anomaly$normal_nrow]
+
+    data_training = list(normal = normal_anomaly$normal[idx_normal_2_training, ])
+
+    data_test = list(normal = normal_anomaly$normal[idx_normal_2_test,],
+                     anoamaly = normal_anomaly$anomaly)
```
Comment (Verntrax) [inline src/main.R]: anomaly

**Review by Verntrax — CHANGES_REQUESTED** [review]

File: src/main.R (hunk 1 above)
Comment (unitatem) [reply src/main.R]: done

File: src/main.R (hunk 2 above)
Comment (unitatem) [reply src/main.R]: done

File: src/main.R (hunk 3 above)
Comment (unitatem) [reply src/main.R]: done

**Review by Verntrax — APPROVED** [review]

**Pull request merged**